app/assets/stylesheets/publisher.scss

```diff
@@ -29,25 +29,37 @@
     }
 
     .options_and_submit {
-      padding: 10px 0 ;
-      #publisher_service_icons {
-        .btn-link{ text-decoration: none; }
-        .btn-link.question_mark .entypo-cog {
+      padding: 10px 0;
+      .btn{ height: 32px; }
+      #publisher-service-icons {
+        text-decoration: none;
+        .entypo-cog, .service_icon {
           color: $text-grey;
           font-size: 16px;
-          line-height: $line-height-computed;
-        }
-        .btn-link.question_mark:hover.entypo-cog { color: $black; }
-        .btn-link.service_icon {
-          padding-left: 5px;
-          padding-right: 5px;
+          line-height: 16px;
```

Comment:
Yes, but `$line-height-computed` is the line height computed from the default font size which is 13px. If you increase the font size (here 16px) without decreasing the line height, the button will have a bigger height. Try to to unset the 32px height and setting back the line height you will see that this button is higher than the others.